### DISCUSSION TITLE
Declutter Settings page

### DISCUSSION THREAD

**Discussion by markcheret** [markcheret/footnotes#129] [Ideas]
I just had a look at the settings page for footnotes.
Wow. That has exploded since I last thoroughly checked it!!!

Erm... It needs decluttering. At least visually.

Either the postboxes need to be closed by default when opening the settings or we move some (less used and special cases) over to an "Expert Mode" page.
Also, all those explanations need to be either made more end-user readable or hidden and accessible via a mouse-over á la "?" .

I'll post a few screenshot suggestions in a bit, just wanted to kick off the discussion here.

**Comment by markcheret** [comment]
Oh and I'm also happy to try my hand at committing some code :)

**Comment by Rumperuu** [comment]
Yeah, I was going to make an Issue for this actually. As far as I can tell, it's a mix of settings that don't need to be there in the first place (e.g., https://github.com/markcheret/footnotes/issues/48#issuecomment-814193614), settings that have been introduced in response to a single bug report and settings that are good to have but need better presentation. Adding an ‘Advanced Settings’ tab is a good shout.

I'll see what else I come up with during #87 and add it here.

**Comment by Rumperuu** [comment]
Moved to Issue #161
